petri/src/vm/openvmm/mod.rs

```diff
@@ -203,6 +203,12 @@ impl PetriVmConfig for PetriVmConfigOpenVmm {
         Box::new(Self::with_vmbus_redirect(*self))
     }
 
+    fn with_scsi_relay(self: Box<Self>, _: bool) -> Box<dyn PetriVmConfig> {
+        // SCSI relay is not supported for OpenVMM backend
+        // OpenVMM handles VTL2 settings differently
+        self
+    }
+
```

Comment:
1. In general: if something is not supported, then make sure to return an `Err` when called.
2. Specifically here: examine cases where VTL2 settings are used for openvmm; see if you can make the pattern the same between openvmm and powertest based tests